app/steps/ui/tasklist/includes/executors.html

```diff
@@ -1,24 +1,36 @@
 {% set previousTaskStatus = fields.previousTaskStatus.value.ExecutorsTask %}
 
 <li class="task-item">
-    <span class="heading-medium">{{content.executorsTask}}</span>
-    {% if fields.ExecutorsTask.value.status === 'complete' and previousTaskStatus !== 'started' %}
-        <span class="progress-status-tag">{{content.taskCompleteTag | upper}}</span>
-        <p>{{content.executorsTaskCompleteParagraph1}}</p>
+
+    {{taskStatus(
+        content = content,
+        title = content.executorsTask,
+        status = fields.ExecutorsTask.value.status,
+        previousTaskStatus = previousTaskStatus
+    )}}
+
+    {% if fields.ExecutorsTask.value.status === 'complete' %}
+        <div class="task-item-content">
```

Comment:
Can we also move this wrapper `<div>` out of this conditional statement so it's not repeated in each `if`/`else` part of the statement?